### PR TITLE
Air 2080: Implement My Collections using new endpoint

### PR DESCRIPTION
**Pull request by bFraley**
This feature work is mostly complete, may need some additional clean out of unused code, but opening PR to get some eyes on it.

### CODE REVIEW

**Comment by bFraley** [comment]
Additional code removal on this PR is complete.

File: src/app/browse-page/my-collections.component.ts [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
-import { TagsService } from './tags.service'
-import { Tag } from './tag/tag.class'
+import { LockerService } from 'app/_services'
 import { TitleService, AssetSearchService, AuthService, AssetService, FlagService } from '../shared'
 
 @Component({
   selector: 'ang-my-collections',
   templateUrl: 'my-collections.component.pug',
-  styleUrls: [ './browse-page.component.scss' ]
+  styleUrls: ['./browse-page.component.scss', './tag/tag.component.scss']
```
Comment (codiak) [inline src/app/browse-page/my-collections.component.ts]: Are you being lazy here?

Comment (bFraley) [reply src/app/browse-page/my-collections.component.ts]: yes :) will bring the styles into the component

Comment (bFraley) [reply src/app/browse-page/my-collections.component.ts]: @codiak SCSS are now added to browse page styles

**Pull request merged**